app/controllers/lessons_controller.rb

```diff
@@ -47,42 +47,46 @@ def new
   end
 
   def create
-    lesson = Lesson.new
-    lesson.hash_tag = params[:lesson][:hash_tag]
-    lesson.created_at = Time.now
-    lesson.category = params[:lesson][:category]
-    lesson.description = params[:lesson][:description]
-    lesson.approved = true
-    lesson.save
-    cookies["new_lesson_id"] = lesson.id
+    if user_signed_in?
```

Comment:
The rails `.new` action can take params and automatically put together params that are named like database columns. Refactor to use that and then do anything custom that needs it. See this [StackOverflow post](http://stackoverflow.com/questions/5667075/creating-rails-activerecord-model-from-params-hash). Looking at the diff I can see this was not made in this commit, but still worth pointing out.